src/cursor/cur_table.c

```diff
@@ -804,6 +811,30 @@ __wt_table_range_truncate(WT_CURSOR_TABLE *start, WT_CURSOR_TABLE *stop)
     return (ret);
 }
 
+/*
+ * __wt_cursor_bounds_save --
+ *     Save cursor bounds to restore the state.
+ */
+static inline void
+__wt_cursor_bounds_save(WT_CURSOR *cursor, WT_CURSOR_BOUNDS_STATE *state)
+{
+    WT_ITEM_SET(state->lower_bound, cursor->lower_bound);
+    WT_ITEM_SET(state->upper_bound, cursor->upper_bound);
+    state->flags = cursor->flags;
+}
+
+/*
+ * __wt_cursor_state_restore --
+ *     Restore the cursor's external state.
+ */
+static inline void
+__wt_cursor_state_restore(WT_CURSOR *cursor, WT_CURSOR_BOUNDS_STATE *bounds_state)
```

Comment:
I would rename this to `__wt_cursor_bouds_restore` to be aligned with the save function.

Reply:
[0d7073e](https://github.com/wiredtiger/wiredtiger/pull/8265/commits/0d7073eddddb420b84819296e63ad8f41cdf4533)